parser/parser.go

```diff
@@ -284,13 +284,44 @@ func (p PathFragment) String() string {
 	return buf.String()
 }
 
+type ObjectEntry struct {
+	Key   string `@(Ident | QuotedIdent | String)`
```

Comment:
Can we keep to strict json? Supporting ident allows you to omit quotes and quotedident would be backing quotes